cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v3/lifecycle.rst

```diff
@@ -453,8 +453,8 @@ Scaling Workers
 You can query or change the number of instances of a Worker by using the ``instances``
 parameter with HTTP GET or PUT methods::
 
-  GET <base-url>/namespaces/<namespace-id>/apps/<app-id>/workers/<worker-id>/instances
-  PUT <base-url>/namespaces/<namespace-id>/apps/<app-id>/workers/<worker-id>/instances
+  GET <base-url>/namespaces/<namespace>/apps/<app-id>/workers/<worker-id>/instances
```

Comment:
The only reason is we recently standardized on "name" and "description" for all entities across CDAP. We should probably make everything [entity]-name?